tempesta_fw/sock_srv.c

```diff
@@ -307,18 +311,11 @@ tfw_srv_conn_release(TfwSrvConn *srv_conn)
 	 * in deferred context after a short pause (in a timer
```

Comment:
By the way, `tfw_srv_conn_release()` is called only in this file, so please make it static and remove it's declaration from `server.h`.

Reply:
Done.